MatrixSDK/MXSession.m

```diff
@@ -1338,6 +1345,13 @@ - (void)releasePreventPause
     if (self.preventPauseCount > 0)
     {
         self.preventPauseCount--;
+#if TARGET_OS_IPHONE
+        if (self.preventPauseCount == 0
+            && self.applicationStateService.applicationState == UIApplicationStateBackground)
```

Comment:
Instead of checking equal to background, can you check against not equal to active?